HIP/hip-752.md

```diff
@@ -0,0 +1,482 @@
+---
+hip: 752
+**title: Partitioning Functionality for Fungible Tokens on Hedera**
+author: Sam Wood <sam.wood@toko.network>, Stephanie Yi <stephanie.yi@toko.network>, Eleonora Odorizzi <eleonora.odorizzi@toko.network>, Vicky Lio <vicky.lio@toko.network>
+type: Standards Track
```

Comment:
We have different categories of Standard Track. This hip would fall under the Service category like this:
```suggestion
type: Standards Track
category: Service
```